osu.Game/Database/BeatmapDatabase.cs

```diff
@@ -28,6 +28,11 @@ public class BeatmapDatabase : Database
         // ReSharper disable once NotAccessedField.Local (we should keep a reference to this so it is not finalised)
         private BeatmapIPCChannel ipc;
 
+        /// <summary>
+        /// A default representation of a WorkingBeatmap to use when no beatmap is available.
+        /// </summary>
+        public WorkingBeatmap DefaultBeatmap { get; set; }

```

Comment:
Is it possible to make this readonly somehow? And pass in the background texture to it or async-load it (might require changes at framework level)? Don't really like having to refer to this by `DefaultBeatmap?. ...`